tool/lib/model.dart

```diff
@@ -17,13 +17,24 @@ class DevToolsRepo {
   /// This returns the DevToolsRepo instance based on the current working
   /// directory.
   ///
-  /// This can fail and return null if the current working directory is not
-  /// contained within a git checkout of DevTools.
+  /// This will return null if the current working directory is not
+  /// contained within a git checkout of DevTools. Use [requireInstance] for a
+  /// descriptive exception in this case.
   static DevToolsRepo? getInstance() {
     final repoPath = _findRepoRoot(Directory.current);
     return repoPath == null ? null : DevToolsRepo._create(repoPath);
   }
 
+  /// This returns the DevToolsRepo instance based on the current working
+  /// directory.
+  ///
+  /// Throws if the current working directory is not contained within a git
+  /// checkout of DevTools.
+  static DevToolsRepo requireInstance() {
+    return getInstance() ??
+        (throw Exception('devtools_tool must be run from a DevTools checkout'));
```

Comment:
```suggestion
        (throw Exception('devtools_tool must be run from inside of the DevTools repository directory'));
```

Reply:
How do you suggest changes like this? People do it to my PRs all the time and I've never been able to find the UI to do it 🙃